net/developer-guide/manage-presentation/convert-presentation/convert-powerpoint/convert-powerpoint-to-markdown/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Convert PowerPoint to Markdown in C#
+type: docs
+weight: 140
+url: /net/convert-powerpoint-to-markdown/
+keywords: "Convert PowerPoint to Markdown, Convert ppt to md, PowerPoint, PPT, PPTX, Presentation, Markdown, C#, Csharp, .NET, Aspose.Slides"
+description: "Convert PowerPoint to Markdown in C#"
+---
+
+{{% alert color="info" %}} 
+
+Support for PowerPoint to markdown conversion was implemented in [Aspose.Slides 23.7](https://docs.aspose.com/slides/net/aspose-slides-for-net-23-7-release-notes/).
+
+{{% /alert %}} 
+
+## **Convert PowerPoint to Markdown**
+
```

Comment:
Hi @EmSeriki ,

Please add the following information (as an "important note", or something like this).

It needs to be upfront and catch the users' attention so that the export behavior doesn't look unexpected:

Export to markdown is by default *without images*. If you want to export a document with images, you need to set "ExportType = MarkdownExportType.Visual" and set the BasePath where the images referenced in the markdown document will be saved.